ruby/rails/Gemfile

```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
-ruby "2.1.7"
+ruby "1.9.3"
 
 # tracing
-gem 'ddtrace', '0.4.0'
+gem 'ddtrace', '0.6.0'
```

Comment:
can you point the repo to a stable commit? like:
```ruby
gem 'ddtrace', :git => 'https://github.com/DataDog/dd-trace-rb', :ref => '0bd839d'
```
(changing `0bd839d` with the right one)

Actually this application is deployable and I can deploy it if it points to a right `ddtrace` gem. Also can you change the [ruby version][1] in the file? It's used to build the right Docker container for the deploy.

[1]: https://github.com/DataDog/trace-examples/blob/af9da05de01cee1ea18007f1b6e6b7e956d045f6/ruby/rails/.ruby-version